app/2.0/start/toolbox/set-up.md

```diff
@@ -35,11 +39,11 @@ Polymer CLI is an all-in-one command line tool for Polymer projects. In this tut
     If you're missing any of these dependencies, follow the instructions in
     the following section from the Polymer CLI guide to learn how to install
     each one: [Install section from Polymer
-    CLI guide](/{{{polymer_version_dir}}}/docs/tools/polymer-cli#install).
+    CLI guide]({{{polymer_version_dir}}}/docs/tools/polymer-cli#install).
```

Comment:
Meaning the line `CLI guide]({{{polymer_version_dir}}}/docs/tools/polymer-cli#install)` needs to be:

`CLI guide](/{{{polymer_version_dir}}}/docs/tools/polymer-cli#install)`

Note leading slash.